monitoring/rid_qualifier/test_executor.py

```diff
@@ -0,0 +1,34 @@
+from monitoring.rid_qualifier.aircraft_state_replayer import TestHarness, TestBuilder
+import asyncio
+import arrow
+from monitoring.rid_qualifier.utils import RIDQualifierTestConfiguration, RIDQualifierUSSConfig
+
+def build_uss_config(injection_base_url:str, allocated_track:int = 0) -> RIDQualifierUSSConfig:
+  return RIDQualifierUSSConfig(injection_base_url=injection_base_url, allocated_flight_track_number = allocated_track)
+  
+
+def build_test_configuration(locale: str, auth_spec:str, uss_config:RIDQualifierUSSConfig) -> RIDQualifierTestConfiguration: 
+    now = arrow.now()
+    test_start_time = now.shift(minutes=3) # Start the test three minutes from the time the test_exceutor is run. 
+    
+    test_config = RIDQualifierTestConfiguration(
+      locale = locale,
+      now = now.isoformat(),
+      test_start_time = test_start_time.isoformat(),
+      auth_spec = auth_spec,
+      usses = [uss_config]
+    )
+
+    return test_config
+    
+async def main(test_configuration: RIDQualifierTestConfiguration):
+    # This is the configuration for the test.
+    my_test_builder = TestBuilder(test_configuration = test_configuration)
+    test_payloads = my_test_builder.build_test_payloads()
+    
+    my_test_harness = TestHarness(auth_spec = test_configuration.auth_spec, injection_base_url = test_configuration.usses[0].injection_base_url)
+    
+    my_test_harness.submit_payloads_async(test_payloads=test_payloads)
+    # TODO: call display data evaluator to read RID system state and compare to expectations
```

Comment:
Before marking this PR as ready for review, let's make sure that this line is reached successfully without an exception (and therefore the whole program does not have an error) when invoked using `run_locally.sh` or equivalent.

Reply:
I have now tested codebase vs the mock, screenshot attached below 

![image](https://user-images.githubusercontent.com/1588744/118973915-1b434f00-b96a-11eb-88a8-f137cecb8cb1.png)

Reply:
What codebase are you using that produces the above result?  `test_executor.py` does not have any executable content in it (no `if __name__ == "__main__":` check like `rid_qualifier_entry.py`).  Also, the above command does not reproduce the operations in `run_locally.sh`.

Reply:
You are correct, I have snippet that I use to do local testing (and avoid dealing with potential Docker issues as well), I will now test via `run_locally.sh` as well. 


![image](https://user-images.githubusercontent.com/1588744/119134316-e819c200-ba34-11eb-9276-38b91275776e.png)

Reply:
Just a update, when I use it via the console, it works, I am looking into run_locally.sh, I have tried the --link command in docker run but it is not working. 
![image](https://user-images.githubusercontent.com/1588744/119164521-6df93580-ba54-11eb-9ec4-ed3243692c3a.png)